aten/src/ATen/core/jit_type.h

```diff
@@ -1950,6 +1951,28 @@ struct CAFFE2_API AnyTupleType : public Type {
   : Type(TypeKind::AnyTupleType) {}
 };
 
+// the common supertype of all lists,
+// List[T] <: AnyList for all T
+struct AnyClassType;
+using AnyClassTypePtr = std::shared_ptr<AnyClassType>;
+struct CAFFE2_API AnyClassType : public Type {
+  static AnyClassTypePtr create() {
+    return AnyClassTypePtr(
+        new AnyClassType()); // NOLINT(modernize-make-shared)
+  }
+  bool operator==(const Type& rhs) const override {
+    return rhs.kind() == kind();
+  }
+  std::string str() const override {
```

Comment:
Doesn't this also need a `python_str` for schema-mismatch errors?

Reply:
no, python_str is defaulted to `str()` if it's not defined